kotlin/android/app/src/main/java/dev/firezone/android/core/data/Repository.kt

```diff
@@ -88,6 +91,16 @@ internal class Repository
                 .putString(DEVICE_ID_KEY, value)
                 .apply()
 
+        fun getDisabledResources(): Set<String> {
```

Comment:
```suggestion
        fun getDisabledResourcesSync(): Set<String> {
```

I don't understand Android / Kotlin's async but this should match the other method names, right?